src/screens/TopicPageScreen/elements/TopicDomainHeader.js

```diff
@@ -93,13 +94,13 @@ const styles = StyleSheet.create({
     fontSize: normalizeFontSize(12),
     fontFamily: fonts.inter[400],
     textAlign: 'left',
-    color: isHeaderHide ? colors.white : colors.blackgrey
+    color: isHeaderHide ? COLORS.white : COLORS.blackgrey
   }),
   seeMemberText: () => ({
     fontSize: normalizeFontSize(12),
     fontFamily: fonts.inter[500],
     textAlign: 'left',
-    color: colors.signed_primary,
+    color: COLORS.blue,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color value for `seeMemberText` has been changed from `colors.signed_primary` to `COLORS.blue`. Ensure that this change is intentional and aligns with the overall theme of your application. If `COLORS.blue` and `colors.signed_primary` are not the same, this could potentially lead to inconsistencies in your UI.
```diff
-    color: colors.signed_primary,
+    color: COLORS.blue,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->